File: C/Program18.c
/////////////////////////////////////////////////////////////////////////
//                                                                     //
//  File name :     Program18.c                                        //
//  Description :   Used to Display Sum of Factors of given number     //
//  Author :        Pallavi Omprakash Malewar                          //
//  Date :          19/07/2025                                         //
//                                                                     //
/////////////////////////////////////////////////////////////////////////

#include<stdio.h>

int SumFactors(int iNumber)
{
    int iCount = 0 ;
    int iSum = 0;

    if(iNumber < 0)             // Updator
    {
        iNumber = -iNumber;
    }

    for( iCount = 1 ; iCount <= (iNumber / 2) ; iCount++)
    {
        if((iNumber % iCount) == 0 )
        {
            iSum = iSum + iCount;
        }
    }
    return iSum;
}

int main()
{
    int iValue = 0 , iRet = 0 ;

    printf("Enter Number : \n");
    scanf("%d" , &iValue);

    iRet = SumFactors(iValue);

    printf("Addition of Factors : %d\n" , iRet);

    return 0;

}

File: JAVA/Program18.java
//////////////////////////////////////////////////////////////////////////////////////////
//                                                                                      //
//  File name :     Program18.java                                                      //
//  Description :   Used to Reverse the given Number                                    //
//  Author :        Pallavi Omprakash Malewar                                           //
//  Date :          25/08/2025                                                          //
//                                                                                      //
//////////////////////////////////////////////////////////////////////////////////////////


import java.util.Scanner;

class Digits
{
    public int Reverse(int iNo)
    {
        int iDigit = 0, iRev = 0;

        if(iNo < 0)
        {
            iNo = -iNo;
        }

        while(iNo != 0)
        {
            iDigit = iNo % 10;

            iRev = iRev * 10 + iDigit;

            iNo = iNo / 10;  
        }

        return iRev;
        
    }
}

class Program18
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter Number : ");
        int iValue = sobj.nextInt();

        Digits dobj = new Digits();

        int iRet = dobj.Reverse(iValue);

        System.out.println("Reversed Number : " + iRet);

    }
}